spec/models/post_mover_spec.rb

```diff
@@ -201,6 +201,12 @@
           expect(topic.closed).to eq(true)
         end
 
+        it "moving all posts will create a 'topic is closed' moderator post" do
+          new_topic = topic.move_posts(user, [p1.id, p2.id, p3.id, p4.id], title: "new testing topic name", category_id: category.id)
+
+          expect(topic.posts).to include(have_attributes(post_type: Post.types[:small_action], action_code: "closed.enabled"))
```

Comment:
But the topic is not the last post here, the last post is the "4 posts were split to a new topic..." moderator post. Since I don't think we care about the specific order moderator posts are created in, I think it's fine.